src/world/area_kzn/kzn_01/kzn_01_1_main.c

```diff
@@ -0,0 +1,48 @@
+#include "kzn_01.h"
+
+extern EvtScript N(EVS_Main);
+extern NpcGroupList N(DefaultNPCs);
+
+EntryList N(Entrances) = {
+    [kzn_01_ENTRY_0]    { -445.0,    0.0,    0.0,   90.0 },
+    [kzn_01_ENTRY_1]    {  425.0, -255.0,    0.0,  270.0 },
+};
+
+MapSettings N(settings) = {
+    .main = &N(EVS_Main),
+    .entryList = &N(Entrances),
+    .entryCount = ENTRY_COUNT(N(Entrances)),
+    .tattle = { MSG_MapTattle_kzn_01 },
+};
+
+#include "world/common/atomic/kzn_SmokeTexPanners.inc.c"
+
+EvtScript N(EVS_ExitWalk_jan_22_2) = EVT_EXIT_WALK(60,  kzn_01_ENTRY_0, "jan_22",  jan_22_ENTRY_2);
```

Comment:
Will need to do this across all files
```suggestion
EvtScript N(EVS_ExitWalk_jan_22_2) = EVT_EXIT_WALK(60, kzn_01_ENTRY_0, "jan_22", jan_22_ENTRY_2);
```

Reply:
No problem.